Index: src/main/java/org/example/Field.java
===================================================================
diff --git a/src/main/java/org/example/Field.java b/src/main/java/org/example/Field.java
--- a/src/main/java/org/example/Field.java	
+++ b/src/main/java/org/example/Field.java	
@@ -5,8 +5,8 @@
 
     Helper helper = new Helper();
 
-    public String getField(String File, int lineNo){
-        field = helper.(File, lineNo);
+    public String getField(int row, int column){
+        field = helper.getFieldData(row, column);
         return field;
     }
 }
Index: src/main/java/org/example/Gameboard.java
===================================================================
diff --git a/src/main/java/org/example/Gameboard.java b/src/main/java/org/example/Gameboard.java
--- a/src/main/java/org/example/Gameboard.java	
+++ b/src/main/java/org/example/Gameboard.java	
@@ -10,7 +10,7 @@
     }
 
     public void createGameboard (){
-        for (int i = 0; i < field.length; i++) {
+        for (int i = 0; i < fields.length; i++) {
             fields[i] = field.getField("_Fields", i);
         }
     }
